vscode/README.md

```diff
@@ -331,3 +331,17 @@ When `rubyLsp.formatter` is set to `auto`, Ruby LSP tries to determine which for
 
 If the bundle has a **direct** dependency on a supported formatter, such as `rubocop` or `syntax_tree`, that will be used.
 Otherwise, formatting will be disabled and you will need add one to the bundle.
+
+Here's a code example:
+
+```ruby
+# Gemfile
+group :development do
+  # You can uncomment one of the following to get either rubocp or syntax_tree formatter enabled.
```

Comment:
Since this is an example, I think it's enough to just say `gem "rubocop"` and remove the comment.